static/docs/get-started/install.md

```diff
@@ -1,8 +1,8 @@
 # Install
 
 There are a few recommended ways to install DVC: OS-specific package/installer,
-`pip`, `conda` (Anaconda), and Homebrew. (Depending on your OS, some of these
-may be not available.)
+`pip`, `conda` (Anaconda), and Homebrew (Depending on your OS, some of these may
+be not available).
```

Comment:
Please revert, previous syntax wasn't wrong AFAIK. Full sentence ends, then full sentence between parentheses. (We use this form in many places.) If anything, maybe just get rid of the parentheses altogether? 2nd sentence is relevant enough.

Reply:
An entire sentence can't be in parentheses

Reply:
Why not? Do you have a grammar reference? e.g. https://english.stackexchange.com/questions/78173/entire-sentence-in-parentheses 🙂

Reply:
It might be technically ok but it's not a thing people write...i don't feel particularly strong either way so ill revert it

Reply:
Thanks. You're probably right but we already use and abuse this form 😋 so I guess it will just be a little part of the personality of the docs.